src/components/Beta/TransactionCompleted/index.tsx

```diff
@@ -42,9 +42,11 @@ const TransactionCompleted = ({
         )}
       </Box>
       {isShowButtton && (
-        <Button variant="primary" onClick={onButtonClick}>
-          {buttonText}
-        </Button>
+        <Box width="100%" paddingX={10} mb={10}>
+          <Button variant="primary" onClick={onButtonClick}>
+            {buttonText}
+          </Button>
+        </Box>
```

Comment:
this change is not tested properly. due to this change other places are affected.

![Screenshot 2022-07-06 at 4 16 12 PM](https://user-images.githubusercontent.com/6604146/177533399-6f4d875a-6494-4e65-9eb5-fec760e60efa.png)

Remove liquidity button now has more horizontal padding around.

This type of change needs to be handled in component where TransactionCompleted is used. For e.g. in this case it will be `src/pages/Beta/Pool/EarnWidget/AddLiquidity/ConfirmPoolDrawer/index.tsx`